x-pack/legacy/plugins/siem/public/components/embeddables/actions/apply_siem_filter_action.test.tsx

```diff
@@ -91,15 +86,14 @@ describe('ApplySiemFilterAction', () => {
       const embeddable = {
         type: MAP_SAVED_OBJECT_TYPE,
       };
-      const triggerContext = {};
-      if (isEmbeddable(embeddable) && isTriggerContext(triggerContext)) {
+      if (isEmbeddable(embeddable)) {
+        // @ts-ignore
         const result = await action.isCompatible({
```

Comment:
It is not recommended to use `@ts-ignore`, if there is some TypeScript type mismatch you can cast to specific type or cast to `any`.

```ts
(action as any).isCompatible
(action.isCompatible as any)
(action as SomeType).isCompatible
```